commit 3fa783cc483486a2c7e4cd1ad3612a010a20e2a6 (refs/changes/24/1124/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-12-14T15:55:20-08:00 (2 years, 10 months ago)
    
    joyent/node-smartdc-auth#11 privateKeySigner doesn't allow opts.key to be a buffer

diff --git a/lib/index.js b/lib/index.js
index a9bedb3..0181835 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -80,7 +80,6 @@ function privateKeySigner(options) {
 		throw (new Error('options.key (a String or Buffer) is ' +
 		    'required'));
 	}
-	mod_assert.string(options.key, 'options.key');
 	mod_assert.string(options.user, 'options.user');
 	mod_assert.optionalString(options.subuser, 'options.subuser');
 	mod_assert.optionalString(options.keyId, 'options.keyId');
